keras/regularizers.py

```diff
@@ -241,7 +241,7 @@ def __call__(self, x):
     if self.l1:
       regularization += self.l1 * tf.reduce_sum(tf.abs(x))
     if self.l2:
-      regularization += self.l2 * tf.reduce_sum(tf.square(x))
+      regularization += 2.0 * self.l2 * tf.nn.l2_loss(x)
```

Comment:
IMO `2 * tf.nn.l2_loss(x)` hinders readability compared to `tf.reduce_sum(tf.square(x))`. You don't understand where the 2 comes from and you have to know about an obscure TF API. Basically you cannot understand the code without reading the TF docs.

Reply:
You're right. So I added comments in the code: https://github.com/szutenberg/keras/commit/23a9149a8da92767b18f1e0e02de99d60e4e602f.